### PR TITLE
avoid double logging from healing

### PR DESCRIPTION
**Pull request by harshavardhana**
## Community Contribution License
All community contributions in this pull request are licensed to the project maintainers
under the terms of the [Apache 2 license] (https://www.apache.org/licenses/LICENSE-2.0). 
By creating this pull request I represent that I have the right to license the 
contributions to the project maintainers under the Apache 2 license.

## Description
avoid double logging from healing

## Motivation and Context
less logs

## How to test this PR?
well if you have a healing situation it would 
end up double logging. 

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Unit tests added/updated
- [ ] Internal documentation updated
- [ ] Create a documentation update request [here](https://github.com/minio/docs/issues/new?label=doc-change,title=Doc+Updated+Needed+For+PR+github.com%2fminio%2fminio%2fpull%2fNNNNN)

### CODE REVIEW

**Review by vadmeste — COMMENTED** [review]
One comment

**Comment by harshavardhana** [comment]
> One comment

Don't see your comment @vadmeste

**Review by krisis — APPROVED** [review]

File: cmd/erasure-healing-common.go [hunk 1]
```diff
@@ -347,13 +347,6 @@ func disksWithAllParts(ctx context.Context, onlineDisks []StorageAPI, partsMetad
 					dataErrs[i] = errFileCorrupt
 					continue
 				}
-
-				// Since erasure.Distribution is trustable we can fix the mismatching erasure.Index
```
Comment (vadmeste) [inline cmd/erasure-healing-common.go:351]: @harshavardhana is this intentional ?

Comment (harshavardhana) [reply cmd/erasure-healing-common.go:351]: yes @vadmeste this situation cannot be fixed via healing, The issue is similar to EC:0 to EC:4 - you need to use existing shards to rewrite the incorrect positions.

This is not possible to do in any manner while reading a shard and overwriting the same shard.

**Pull request merged**